### ISSUE TITLE
Missing previous log details

### ISSUE THREAD

**Issue by robnzh** [cgeo/cgeo#6335]
##### Detailed steps causing the problem:
- Step 1 Opened cache GC49JHQ from Gmail on Android phone. 
- Step 2 Entered a disabled log.
- ...

##### Actual behavior after performing these steps:
Step 1:  log page showed cache has no logs (there are in fact a number of logs on this cache.)
Step 2:  Only log showing after log entry was the submitted disabled log.  (note this was checked both from both email link and opening cache details directly in c:geo)

##### Expected behavior after performing these steps:
Step 1: should have shown a number of logs
Step 2: ditto

##### Version of c:geo used:
2017.02.07 

##### Is the problem reproducible for you?
No  (it was but I wasn't able to submit this report at the time, logs are OK a few hours later.)

##### System information:
```
Attach system information here if available (see c:geo menu -> About c:geo -> System)
Keep the apostrophe at beginning and end to have it properly formatted
```
There is no system option under about c:geo, it gives version number as above, has the word "carnero" after the word "lazier" and a donate development button.

##### Other comments and remarks:
Nil

**Comment by robnzh** [comment]
sorry forgot to mention that I am a beta tester but it appears this is the current rather than a beta version.

**Comment by rsudev** [comment]
It might have been a temporary glitch in the communication with groundspeak (server not responding).
Regarding the system information: In 'about c:geo' you have several tabs/pages. 'System' is the fourth page to the right (after 'Help' and 'Changes')

**Comment by Lineflyer** [comment]
@robnzh Is this happening only in case you open a cache from mail link or also if you refresh a cache within c:geo?

**Comment by robnzh** [comment]
Both.  I initially opened it from an email link.   I then tried opening it
from c:geo, still the same error so I refreshed it and still the same
error.  I then submitted the disabled log from c:geo and the only log
showing was that log although there are a number of previous logs.

I did get the same error on another cache a few days earlier but
unfortunately didn't record it. So far I have not had the same problem
recur but if it does I will let you know.

Robert.
